content/docs/learning-guide/tutorials/assets/physx-colliders.md

```diff
@@ -0,0 +1,162 @@
+
+9. **Left-click + drag** the source asset from Asset Browser into the viewport.
+
+    {{< image-width "/images/learning-guide/tutorials/assets/physx-entity.png" "800" "Drag the mesh asset into the viewport">}}
+
+    When you drag the asset into the viewport, O3DE automatically creates an entity with a **Mesh** component that references the mesh product asset. If the source asset contains materials that have been processed, the materials are applied to the mesh by default. Note the `.pxmesh` product asset has been generated and appears beneath the source asset in Asset Browser.
+
+8. Add a **PhysX Collider** component to the entity. With the entity selected, in **Entity Inspector**, choose **Add Component** and select **PhysX Collider** from the component list. The `.pxmesh` asset is automatically detected by the component. The PhysX Collider component's **Shape** property is set to `PhysicsAsset` and the **PhysX Mesh** property automatically references the `.pxmesh` product asset.
+
+9. With just a PhysX Collider component, this is a static entity. PhysX objects can collide with this entity, but the entity will not move or react to PhysX collisions or forces.
+
+{{< note >}}
+If you intend for the entity to be static, you should also activate the **Static** property in the **Transform** component of the entity. The **Static** property optimizes the entity, ensuring the best runtime performance for static entities.
+{{< /note >}}
+
+10. If you intend to create a kinematic or dynamic entity, you must add a **PhysX Rigid Body** component as well. With the entity selected, in Entity Inspector, choose **Add Component** and select PhysX Rigid Body from the component list. Ensure the **Static** property in the entity's Transform component is not active.
+
+{{< note >}}
+Adding the PhysX Rigid Body component makes the entity dynamic. If you choose the {{< icon simulate-physics.svg >}} simulation button, the entity will drop with gravity.
+
+If you want to create a kinematic entity that is animated through script, activate the **Kinematic** property in the PhysX Rigid Body component.
+{{< /note >}}
+
+## Decompose input meshes
+
+Assets that are composed of multiple meshes, or assets that have complex meshes, might require similarly complex PhysX collider assets. This is particularly true for kinematic and dynamic entities where triangle mesh colliders can't be used. In these scenarios, the input mesh can be decomposed into approximately convex parts, and primitive colliders or convex colliders can be generated and automatically fit to each part.
+
+{{< note >}}
+Mesh decomposition doesn't alter the input mesh. Mesh decomposition is part of the process of generating and fitting collider assets. The decomposed mesh is discarded when the collider process job is completed. Mesh decomposition can only be used with primitive or convex colliders.

```

Comment:
I'm not sure what is meant by "the decomposed mesh is discarded". I think the second sentence could be clearer if it emphasized that it's the output that can be primitive or convex.

Reply:
Is this better?

Assets that are composed of multiple meshes, or assets that have complex meshes, might require similarly complex PhysX collider assets. This is particularly true for kinematic and dynamic entities where triangle mesh colliders can't be used. In these scenarios, the input mesh can be decomposed into convex parts. Primitive colliders or convex colliders can be automatically generated and fitted to each part and processed as collider `.pxmesh` product assets.